helpers/context.py

```diff
@@ -127,6 +130,57 @@ async def on_timeout(self):
             await self.message.delete()
 
 
+class RequestView(discord.ui.View):
```

Comment:
Can this be a subclass of something?

Reply:
Done, does it look ok?

Reply:
Note: I added the `delete_after_timeout` parameter because previously the trade and battle reaction requests would not delete after timing out. So this parameter is False for these requests while True for other confirmations to stay true to the original behaviour.

Reply:
Is there a way to merge `ctx.confirm` and `ctx.request` perhaps? And pass in the RequestView class as cls for requests? I tried but I was unable to figure out a good solution since it needs an extra parameter `requestee`